src/py21cmfast/wrapper.py

```diff
@@ -1459,6 +1459,7 @@ def __init__(
         perturb: PerturbedField,
         ib: IonizedBox,
         bt: BrightnessTemp,
+        st: [TsBox, None],
         photon_nonconservation_data=None,
     ):
         _check_compatible_inputs(init_box, perturb, ib, bt, ignore=[])
```

Comment:
You don't check for compatible inputs for `st`. Is this because you are assuming that inputs will be the same?

Reply:
Nope, should check for that, thanks!

Reply:
It's now checking for this.